.markdownlint.json

```diff
@@ -0,0 +1,3 @@
+{
+    "MD013": false
```

Comment:
NIT: can we add a comment here explaining what does this code mean?

Reply:
Json files should not includes comments :-) But I'll add it some documentation. I'll add a page dedicated to help our developer dealing with Markdown linter.
What do you think if I add this one I've created for nanoFramework in there as well (I'll adjust it for us of course)?
https://github.com/nanoframework/nanoframework.github.io/blob/pages-source/content/contributing/markdown-creation.md